app/views/fagus/categories/index.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.array! @categories do |category|
+  json.id category.id
+  json.name category.name
```

Comment:
In that case, you should probably remove the file altogether and add this in later if needed.